api/core/tools/provider/builtin/wecom/tools/wecom_group_bot.yaml

```diff
@@ -0,0 +1,40 @@
+identity:
+  name: wecom_group_bot
+  author: Bowen Liang
+  label:
+    en_US: Send message to Wecom group bot
```

Comment:
The label is a little longer, it's better to make it shorter and iconic, maybe "SendGroupMessage" ?
<img width="565" alt="image" src="https://github.com/langgenius/dify/assets/45712896/473fc69b-b087-44d5-92ec-8ed29840474c">

Reply:
Thanks. Shorted the labels.